java/com/google/security/wycheproof/testcases/EciesTest.java

```diff
@@ -294,7 +294,7 @@ public void testIsAlias(String algorithmA, String algorithmB) throws Exception {
     byte[] message = "Hello".getBytes("UTF-8");
     eciesA.init(Cipher.ENCRYPT_MODE, keyPair.getPublic());
     byte[] ciphertext = eciesA.doFinal(message);
-    eciesB.init(Cipher.DECRYPT_MODE, keyPair.getPrivate(), eciesB.getParameters());
+    eciesB.init(Cipher.DECRYPT_MODE, keyPair.getPrivate(), eciesA.getParameters());
```

Comment:
@thaidn you swapped `eciesA.getParameters()` to `eciesB.getParameters()` in 7dfe46b. In BouncyCastle 1.59 this causes the test to fail since `eciesB.getParameters()` returns `null`, therefore `nonce` is `null` and `InvalidAlgorithmParameterException` is thrown. The test passes by swapping them, however I'm unsure if that changes the meaning of the test.

https://github.com/bcgit/bc-java/blob/master/prov/src/main/java/org/bouncycastle/jcajce/provider/asymmetric/ec/IESCipher.java#L295